docs/features-timeline.md

```diff
@@ -0,0 +1,219 @@
+---
+id: features-timeline
+title: Features Timeline
+---
+
+
+<ol class="timeline-container">
+<li data-date="Feb 2021">
+
+## Babel 7.13.0
+
+- Top-level `targets` option ([RFC](https://github.com/babel/rfcs/pull/2))
+- Granular compiler assumptions ([docs](https://babeljs.io/assumptions), [RFC](https://github.com/babel/rfcs/pull/5))
+- Support for the [Records and Tuples](https://github.com/tc39/proposal-record-tuple) proposals
+  ```js
+  let rec = #{"{"} x: 1 {"}"};
+  let tup = #[1, 2, 3];
+  ```
+- [TypeScript 4.2](https://devblogs.microsoft.com/typescript/announcing-typescript-4-2/) support
+
+</li>
+<li data-date="Oct 2020">
+
+## Babel 7.12.0
+
+- Support for the [class static blocks](https://github.com/tc39/proposal-class-static-block) proposal
+  ```js
+  class A {"{"}
```

Comment:
oh I guess in the conversion these all got wrapped!

Reply:
Oh thanks 😅